app/src/main/java/org/mozilla/tv/firefox/navigationoverlay/NavigationOverlayFragment.kt

```diff
@@ -276,6 +277,11 @@ class NavigationOverlayFragment : Fragment() {
                     AccountState.AuthenticatedNoProfile -> {
                         fxaButton.setImageResource(R.drawable.ic_avatar_authenticated_no_picture)
                         fxaButton.contentDescription = resources.getString(R.string.fxa_navigation_item_signed_in2)
+
+                        val settings = Settings.getInstance(context!!)
+                        if (settings.shouldShowFxaOnboarding()) {
+                            navigationOverlayViewModel.showFxaOnboardingScreen(context!!)
```

Comment:
It looks like this code should work to me, but I created a new FF account, logged in here, then verified my account, and I didn't get the onboarding popup.  I don't have any account where I can test [this expected behavior](https://github.com/mozilla-mobile/firefox-tv/issues/2496#issuecomment-528434160), were you able to verify that it works as expected?

Reply:
This is working as expected for me, weird :/